File: Python/function.py
#!/usr/bin/env python

# Function to calculate the square of a number
def square(x):
    return x*x

# Function to double a number
def doubl(x):
    return x*2

# Get a number from the user
question = 'Enter a number: '
number = input(question)

# Output
print 'The square of ', number, ' is ', square(number)
print number, 'doubled is ', doubl(number)

File: C/function.c
/*
 * Demonstrate C Functions
 *
 * Compile with: gcc -o function function.c
 *
 */

#include <stdio.h>

/*
 * Define a function to square a number
 */
int square(int number){
    int retval = number * number;
    return retval;
}

/*
 * Define a function to double a number
 */
int doubl(int number){
    int retval = number * 2;
    return retval;
}

/*
 * Main function
 */
int main(void){
    int number;

    /* Ask for a number */
    printf("Enter a number: "); 
    scanf("%d", &number);

    /* Output computed values */
    printf("The square of %d is %d.\n", number, square(number));
    printf("%d doubled is %d\n", number, doubl(number));

    return(0);
}
